docs/source/frameworks/index.rst

```diff
@@ -48,44 +59,46 @@ repository.
         :link: /frameworks/xgboost
         :link-type: doc
 
-    .. grid-item-card:: :doc:`/frameworks/fastai`
-        :link: /frameworks/fastai
-        :link-type: doc
 
-    .. grid-item-card:: :doc:`/integrations/mlflow`
-        :link: /integrations/mlflow
-        :link-type: doc
+Custom Models
```

Comment:
moved out pickable and custom runner for "custom models"

will add an additional guide for custom ml framework in bentoml later on